### ISSUE TITLE
"Redo" does not work

### ISSUE THREAD

**Issue by treyworley** [hediet/vscode-drawio#328]
In the edit tab of VS Code, "Undo" works to undo the last edit to a diagram; however, "redo" does not work. This is true for both using the dropdown under "Edit" and clicking the buttons as well as using the keyboard shortcuts (on Mac, CMD+Z and CMD+Y).

This functionality is the case on both an Amazon Linux system and Mac.
Linux info:
![image](https://user-images.githubusercontent.com/61593851/182911893-55224487-cb73-4984-b92c-4d772ab704c8.png)

Mac info:
MacBook Pro, 13-inch, M1, 2020
macOS Monterey 12.5

I am primarily using files with `drawio.svg` extension

**Comment by hediet** [comment]
I cannot reproduce...

![recording](https://user-images.githubusercontent.com/2931520/183019230-2c79b343-83cf-4b4f-bf07-ee705f80b977.gif)

**Comment by treyworley** [comment]
Hmm ok, thanks for checking. I can actually use the Redo arrow that is in the Drawio top bar (as you show), just not the VS Code's Edit > Redo (or accompanying shortcut). Does not seems to be the extension, I suppose. Thanks again. Wonder if others have this issue

**Comment by hediet** [comment]
You sure you use the *.svg extension and not *.png?

**Comment by treyworley** [comment]
Yes, I always use *.drawio.svg extensions names. VS Code's Undo/Redo work as expected for text files, notebooks, etc. With a `*drawio.svg` file, VS Code's "Redo" never works and "Undo" will sometimes do something different that the buttons in the Drawio extension's top bar. The extension's "Undo" and "Redo" work as expected and give the desired results. It is mainly annoying since I can't use my keyboard shortcuts for Undo/Redo while in a Drawio file (they are mapped to VS Code's Undo/Redo).
